### PR TITLE
Issue #2 Update JJB for minishift-b2d-iso job

### LINKED ISSUE #2: Update JJB for minishift-b2d-iso job
Similar to [minishift-centos-iso job](https://github.com/minishift/minishift-ci-jobs/blob/master/minishift-ci-index.yaml#L73-L77), create job for `minishift-b2d-iso`.

### PR DESCRIPTION
**Pull request by budhrg**
Fix #2

### CODE REVIEW

File: minishift-ci-index.yaml [hunk 1]
```diff
@@ -75,3 +75,8 @@
           ci_project: '{name}'
           ci_cmd: '/bin/bash centos_ci.sh'
           timeout: '20m'
+      - '{git_repo}':
```
Comment (hferentschik) [inline minishift-ci-index.yaml]: +1 Looks good to me. Maybe we can as part of this pull request add Lala and Preveen to the admin list.

Comment (hferentschik) [inline minishift-ci-index.yaml]: Unrelated question, 'name: atomicapp' - is this what we use initially? Was there not talk about creating a new name/group for us?

**Comment by hferentschik** [comment]
add to whitelist

File: minishift-ci-index.yaml (hunk 1 above)
Comment (budhrg) [inline minishift-ci-index.yaml]: @hferentschik , right now waiting for new a/c, once that is done we can update that. 

Ya will add lala and praveen in admin list too.

**Pull request merged**